
From: William Lee Irwin III <wli@holomorphy.com>

And one more alpha fix for the profiling consolidation:
DESC
From: William Lee Irwin III <wli@holomorphy.com>
Subject: alpha irq needs profile.h for create_prof_cpu_mask()
EDESC

Looks like there's one more piece needed for alpha:

Index: premm2-2.6.8-rc3/arch/alpha/kernel/irq.c
===================================================================

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/alpha/kernel/irq.c |    4 +---
 1 files changed, 1 insertion(+), 3 deletions(-)

diff -puN arch/alpha/kernel/irq.c~proc_pc-alpha-fix arch/alpha/kernel/irq.c
--- 25/arch/alpha/kernel/irq.c~proc_pc-alpha-fix	2004-08-07 19:28:50.305267976 -0700
+++ 25-akpm/arch/alpha/kernel/irq.c	2004-08-07 22:01:43.097791552 -0700
@@ -25,6 +25,7 @@
 #include <linux/irq.h>
 #include <linux/proc_fs.h>
 #include <linux/seq_file.h>
+#include <linux/profile.h>
 
 #include <asm/system.h>
 #include <asm/io.h>
@@ -328,9 +329,6 @@ register_irq_proc (unsigned int irq)
 void
 init_irq_proc (void)
 {
-#ifdef CONFIG_SMP
-	struct proc_dir_entry *entry;
-#endif
 	int i;
 
 	/* create /proc/irq */
_
